src/backup.h

```diff
@@ -0,0 +1,13 @@
+#ifndef BACKUP_H
+#define BACKUP_H
+
+// Backup related functions
+#include <string>
+
```

Comment:
You need to forward declare CWallet here:

`class CWallet;`

Reply:
good catch i hadnt tried a random scenario of putting this in another area without CWallet class. :)